### DISCUSSION TITLE
Help Adding to Project

### DISCUSSION THREAD

**Discussion by RealCrayfish** [ArthurSonzogni/FTXUI#379] [Q&A] [answered]
Hi, I've just discovered this library but have no clue how to add it to my project and I still don't understand after reading the docs. Please can someone help me and tell me what to do?

Im using Visual Studio for my IDE

**Comment by ArthurSonzogni** [comment, marked as answer]
Hello!

What windows users like to use? What do you use usually?

Most C++ folks are using CMake, I dependently of their OS or toolkit. That's what I would recommend.
You can learn how to use CMake and how visual studio integrate CMake. Then look into this example:
https://github.com/ArthurSonzogni/ftxui-starter/blob/master/CMakeLists.txt

I believe you can download ftxui-starter and open the CMakeList.txt directly through it.

Alternatively, I believe visual studio users like using vcpkg. FTXUI is available through it:
https://vcpkg.info/port/ftxui

**Reply by RealCrayfish** [reply]
Thanks! I just tried it out and its working completely fine.
Before I was using just a standard Visual Studio C++ project and the pre-built headers I downloaded from the GitHub were from v2.0. Now with it integrated into CMake its working fine. Thanks again!
